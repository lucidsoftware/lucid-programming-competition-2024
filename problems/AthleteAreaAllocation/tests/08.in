version https://git-lfs.github.com/spec/v1
oid sha256:6712a35cecc265fe7e01acfd18ab851c30bbd365eaf4489ee66fe86ebe7b5cd4
size 47140
