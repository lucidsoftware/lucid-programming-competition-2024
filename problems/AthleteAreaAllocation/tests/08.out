version https://git-lfs.github.com/spec/v1
oid sha256:69d3ac8770361df42a7605c9e1746d12fce0b6dea530652257a140df6bc27f22
size 6
